Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -106,6 +106,26 @@ void AdvancedPane::CreateLayout()
          "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>"));
   timing_group_layout->addWidget(correct_time_drift);
 
+  auto* const rush_frame_presentation =
+      new ConfigBool{tr("Rush Frame Presentation"), Config::RUSH_FRAME_PRESENTATION};
+  rush_frame_presentation->SetDescription(
+      tr("Limit throttling to just once after each frame,"
+         " speeding through emulation to reach presentation,"
+         "<br>displaying sooner, and thus reducing input latency."
```

Comment:
Is this linebreak really necessary?

Reply:
I guess I didn't like super wide tool tips, but I'll remove the weird breaks.